salt_resources/states/basebox/miniconda.sls

```diff
@@ -0,0 +1,9 @@
+download_miniconda_installer:
+  cmd.run:
+    - name: wget http://repo.continuum.io/miniconda/Miniconda-latest-Linux-x86_64.sh -O /home/vagrant/miniconda.sh
```

Comment:
Looked fine at a glance! I noticed that this isn't specifying which miniconda though. This url grabs miniconda2, but not the newest one, and I haven't ran into this link in the conda docs. I think it's deprecated probably. It's not hugely important right now, because it works. It does make me realize I didn't specify in the ticket which miniconda version I wanted. We should probably support both 2 & 3, but that can be in another ticket unless you're feeling generous.